src/Microsoft.Framework.Runtime.Hosting/RuntimeHost.cs

```diff
@@ -1,19 +1,124 @@
-﻿using NuGet.ProjectModel;
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Versioning;
+using NuGet.DependencyResolver;
+using NuGet.Frameworks;
+using NuGet.LibraryModel;
+using NuGet.ProjectModel;
 
 namespace Microsoft.Framework.Runtime.Hosting
 {
     public class RuntimeHost
     {
         public Project Project { get; }
-        public string ApplicationBaseDirectory { get; }
+        public NuGetFramework TargetFramework { get; }
+        public IAssemblyLoaderContainer LoaderContainer { get; }
```

Comment:
This shouldn't be part of the runtime host. We need to separate the runtime from the ability to wire up the dependency providers and get information about the project closure.